scripts/add_content_to_db.py

```diff
@@ -70,17 +70,44 @@
     }
 
     with open(args.csv, mode="r", encoding="utf-8") as file:
+        reader = csv.DictReader(file)
+        tags = [val.upper() for row in reader for val in eval(row["tags"])]
```

Comment:
Ah ok so this expects the `tags` column values to look like `"['tag1', 'tag2']"`. 

Might make sense if we're getting a dump from some DB, but I think often we're getting data from excel/google sheets, and in google sheets, I'd imagine users would write `"tag1, tag2"` instead? (cc: @Tanmay-97  what do you think?)

<img width="156" alt="image" src="https://github.com/IDinsight/aaq-core/assets/7042047/a301af35-3f05-4ae8-a21d-f12c4e218aef">

which ends up with csv file that looks like

```
tags
"tag1,tag2"
"tag2, tag3"
```
In which case we can parse this way instead:
```suggestion
        tags = [val.strip().upper() for row in reader for val in row["tags"].split(",")]
```

Reply:
For the import feature, we should be more robust and probably want to handle both cases

Reply:
I'm thinking just handle the natural TAG1,TAG2 usecase (which is what our export button creates too), not the weird list situation